app/helpers/custom_links_helper.rb

```diff
@@ -1,12 +1,16 @@
 module CustomLinksHelper
-  def custom_links
+  def custom_links(controller)
     require 'json'
     result = ''
     source = CustomSetting['custom_content.header_menus']
     begin
       json = JSON.parse(source, quirks_mode: true)
       json.each do |elm|
-        result += "<li>#{link_to elm['label'], elm['url']}</li>"
+        active_class = ''
+        if (elm['url'].start_with?("#{root_url}pages/") || elm['url'].start_with?('/pages/')) && controller.controller_name == 'pages'
```

Comment:
__[RuboCop]__

Performance/DoubleStartEndWith: Use `elm['url'].start_with?("#{root_url}pages/", '/pages/')` instead of `elm['url'].start_with?("#{root_url}pages/") || elm['url'].start_with?('/pages/')`.


<!-- Comment by [SideCI](https://sideci.com) -->